tox.ini

```diff
@@ -41,31 +45,33 @@ envlist =
 # Kombu >= 4.2 only supports Python 3.7+
     kombu_contrib-py{37,38,39}-kombu{42,43,44,45,46,}
     kombu_contrib-py{310}-kombu{43,44,45,46,}
-    molten_contrib-py{36,37,38,39,310}-molten{06,07,10,}
+    kombu_contrib-py{311}-kombu{50,51,}
+    molten_contrib-py{36,37,38,39,310,311}-molten{06,07,10,}
     mysqldb_contrib-py{27,35,36,37,38,39}-mysqlclient{13,14,}
-    mysqldb_contrib-py{310}-mysqlclient{14,}
+    mysqldb_contrib-py{310,311}-mysqlclient{14,}
     pylibmc_contrib-py{27,35,36,37,38,39,310}-pylibmc{140,150,}
+    pylibmc_contrib-py{311}-pylibmc{160,}
     pylons_contrib-py27-pylons{096,097,010,10,}
     pymongo_contrib-py{27,35,36,37}-pymongo{30,31,32,33,34,35,36,37,38,39,310,}-mongoengine
 # pymongo does not yet support Python 3.8: https://github.com/pymssql/pymssql/issues/586
 # but these tests still work.
-    pymongo_contrib-py{38,39,310}-pymongo{30,31,32,33,35,36,37,38,39,310,}-mongoengine
-    pynamodb_contrib-py{27,35,36,37,38,39,310}-pynamodb{40,41,42,43,}-moto1
+    pymongo_contrib-py{38,39,310,311}-pymongo{30,31,32,33,35,36,37,38,39,310,}-mongoengine
+    pynamodb_contrib-py{27,35,36,37,38,39,310,311}-pynamodb{40,41,42,43,}-moto1
     pyodbc_contrib-py{27,35,36,37,38,39}-pyodbc{3,4}
-    requests_contrib{,_autopatch}-py{27,35,36,37,38,39,310}-requests{208,209,210,211,212,213,219}
-    sqlite3_contrib-py{27,35,36,37,38,39,310}-sqlite3
+    requests_contrib{,_autopatch}-py{27,35,36,37,38,39,310,311}-requests{208,209,210,211,212,213,219}
+    sqlite3_contrib-py{27,35,36,37,38,39,310,311}-sqlite3
     tornado_contrib-py{27,35,36,37,38,39}-tornado{44,45}
     tornado_contrib-py{37,38,39}-tornado{50,51,60,}
-    tornado_contrib-py{310}-tornado{60,}
+    tornado_contrib-py{310,311}-tornado{60,}
     tornado_contrib-py27-tornado{44,45}-futures{30,31,32,}
     vertica_contrib-py{27,35,36,37,38,39}-vertica{060,070}
 # Opentracer
-    py{27,35,36,37,38,39,310}-opentracer
-    py{35,36,37,38,39,310}-opentracer_asyncio
-    py{35,36,37,38,39,310}-opentracer_tornado-tornado{44,45,50,60,}
+    py{27,35,36,37,38,39,310,311}-opentracer
+    py{35,36,37,38,39,310,311}-opentracer_asyncio
+    py{35,36,37,38,39,310,311}-opentracer_tornado-tornado{44,45,50,60,}
     py{27,35,36}-opentracer_gevent-gevent{11,12}
     py{37,38}-opentracer_gevent-gevent{13,14}
-    py{39,310}-opentracer_gevent-gevent{209,2012,211}
+    py{39}-opentracer_gevent-gevent{209,2012,211}
```

Comment:
Why is 3.10 being removed?

Reply:
Add comment explaining gevent compatibility issue with opentracer